compiler/lib/Dialect/mhlo/Transforms/HloSimplify.cpp

```diff
@@ -33,6 +33,25 @@ using namespace mlir;
 
 namespace {
 
+static Value createIntialZeroValue(Type elementTy, PatternRewriter &rewriter,
+                                   Location &loc) {
+  auto constType = RankedTensorType::get({}, elementTy);
+  if (isa<mlir::FloatType>(elementTy)) {
+    auto constAttr = DenseElementsAttr::get(
+        constType,
+        {APFloat::getZero(cast<mlir::FloatType>(elementTy).getFloatSemantics(),
+                          /*negative=*/false)});
+    return rewriter.create<mhlo::ConstantOp>(loc, constType, constAttr);
+  } else if (isa<mlir::IntegerType>(elementTy) &&
+             elementTy.getIntOrFloatBitWidth() != 8) {
```

Comment:
why `!= 8`?

Reply:
> why `!= 8`?

Just copied from other where, it's going to remove this.